appserver/controllers/amp_streams_api_controller.py

```diff
@@ -12,25 +12,25 @@
 
 from util.api_service import ApiService, ApiError
 from util.logger import logger
-from amp4e_events_input.amp_storage_wrapper import AmpStorageWrapper
-from amp4e_events_input.stream_dict_manager import StreamDictManager
 
+from amp4e_events_input_lib.amp_storage_wrapper import AmpStorageWrapper
+from amp4e_events_input_lib.stream_dict_manager import StreamDictManager
+
+def jsonify_errors(func):
```

Comment:
Maybe move this method into a separate file and import

Reply:
Ok makes sense, should I do the same with ` def with_validated_response(func)` in `api_service.py`

Reply:
Ah, actually, since these are only used in one class, what about defining them in an inner class? https://medium.com/@vadimpushtaev/decorator-inside-python-class-1e74d23107f6